Sources/NMeta/Extensions/Request+NMeta.swift

```diff
@@ -13,22 +13,46 @@ public extension Request {
         public let deviceOS: String
         public let device: String
         
+        private enum webEnvironment {
+            static let platform = "web"
+            static let versionString = "0.0.0"
+            static let deviceOS = "N/A"
+            static let device = "N/A"
+        }
+        
         init(request: Request) throws {
             guard let metaString = request.headers.first(name: request.application.nMeta.headerName) else {
                 throw NMetaError.headerMissing
             }
 
             var components = metaString.components(separatedBy: ";")
-
-            guard components.count == 5 else {
+            
+            // As the web environment _does not_ need to have all 5 values we initially checks
+            // that at least two components are present (platform and environment)
+            guard components.count >= 2 else {
                 throw NMetaError.invalidHeaderFormat
             }
-
+            
             self.platform = components.removeFirst()
             self.environment = components.removeFirst()
-            self.version = try Version(string: components.removeFirst())
-            self.deviceOS = components.removeFirst()
-            self.device = components.removeFirst()
+            
+            if self.platform.lowercased() == webEnvironment.platform {
```

Comment:
@pbodsk 
How about this:
- first get the total count: `let componentsCount = components.count` (after line 28) so we don't have to check several times
- add the condition `, componentCount != 5` to this if statement

Then we'd end up in the else clause if 5 components are specified and we'd get all of them instead of just the first 2 🙂 

An alternative to this could be to use `popFirst()` for 3rd to 5th components like so:

```swift
self.version = components.popFirst() ?? try Version(string: webEnvironment.versionString)
self.deviceOS = components.popFirst() ?? webEnvironment.deviceOS
self.device = components.popFirst() ?? webEnvironment.device
```
That way we'd support supplying 2, 3, 4, or 5 components for web.

Reply:
@siemensikkema I tried...oh man did I try 😄 but...lets discuss tomorrow when we're at the same office. There are some quirks hidden